src/etc/poudriere.conf.sample

```diff
@@ -354,3 +354,7 @@ DISTFILES_CACHE=/usr/ports/distfiles
 # be fetched.
 # Default: everything
 #PACKAGE_FETCH_WHITELIST="gcc* rust llvm*"
+
+# Set the base mount point for jails
+# Default:
+#JAIL_BASEMNT=${POUDRIERE_DATA}/.m
```

Comment:
I think we should not put this here as it is not something a user typically needs to modify, and the default is confusing and error-prone since POUDRIERE_DATA can be dynamic / undefined here.

Reply:
Where should this belong to?